src/orochi-network/orand-v1.md

```diff
@@ -0,0 +1,17 @@
+# Orand V1
+
+Orand project was built based on Elliptic Curve Verifiable Random Function (ECVRF), it's deterministic and verifiable based on EC assumption. Orochi Network unable to manipulate the result.
```

Comment:
```suggestion
Orand project was built based on Elliptic Curve Verifiable Random Function (ECVRF). It is deterministic and verifiable based on assumptions from elliptic curves. Administrators of Orochi Network are unable to manipulate the results.
```

Is there any specific assumptions in elliptic curves? I rarely heard about the term ``EC assumption'', so I changed to assumptions in elliptic curves. It would be better if we can precisely mention those specific assumptions, like XDH, SIDH, ...

Orochi Network is a network. I am not sure whether we can say Orochi Network cannot manipulate ... Instead, can you choose some more appropriate word. For me, I am not sure so I temporarily use the word "administrators".